### PR TITLE
PLAT-4124 Added the table attributes "colspan" and "rowspan" to the td element

### PR DESCRIPTION
**Pull request by deepaksubu**

### CODE REVIEW

**Comment by deepaksubu** [comment]
@ldrozdz please review this piece

File: src/main/java/org/symphonyoss/symphony/messageml/elements/TableCell.java [hunk 1]
```diff
@@ -27,11 +28,27 @@
...
 
   public TableCell(Element parent) {
     super(parent, MESSAGEML_TAG);
   }
 
+  @Override
+  void buildAttribute(org.w3c.dom.Node item) throws InvalidInputException {
+    switch (item.getNodeName()) {
+      case ATTR_ROWSPAN:
+        setAttribute(ATTR_ROWSPAN, getStringAttribute(item));
```
Comment (ldrozdz) [inline src/main/java/org/symphonyoss/symphony/messageml/elements/TableCell.java]: We should validate that the spans are numberrs and non-negative. You can do it here by using _getLongAttribute()_ (or, better still, adding a method _getIntAttribute()_) or  in the  _validate()_ method which you'd have to override.

File: src/test/java/org/symphonyoss/symphony/messageml/elements/ElementTest.java [hunk 2]
```diff
@@ -1729,7 +1730,10 @@ public void testTable() throws Exception {
     cell = row.getChildren().get(0);
     assertEquals("Element class", TableCell.class, cell.getClass());
     assertEquals("Element tag name", "td", cell.getMessageMLTag());
-    assertEquals("Element attributes", Collections.emptyMap(), cell.getAttributes());
+    final HashMap<String, String> expectedAttributeMap = new HashMap<>();
+    expectedAttributeMap.put("colspan", "6");
+    expectedAttributeMap.put("rowspan", "7");
+    assertEquals("Element attributes", expectedAttributeMap, cell.getAttributes());
```
Comment (ldrozdz) [inline src/test/java/org/symphonyoss/symphony/messageml/elements/ElementTest.java]: Also assert that the plaintext representation (context.getText()) doesn't contain those attrs by any chance.

Comment (deepaksubu) [reply src/test/java/org/symphonyoss/symphony/messageml/elements/ElementTest.java]: That is already asserted below this code

File: src/main/java/org/symphonyoss/symphony/messageml/elements/TableCell.java (hunk 1 above)
Comment (deepaksubu) [reply src/main/java/org/symphonyoss/symphony/messageml/elements/TableCell.java]: fixed. Please check now

**Comment by deepaksubu** [comment]
@ldrozdz please review

**Comment by ldrozdz** [comment]
ok to test

**Pull request merged**